src/main.cpp

```diff
@@ -73,17 +82,17 @@ void loop() {
         analogWrite(PIN_MOSFET_A_LO, 0);
         analogWrite(PIN_MOSFET_A_HI, 0);
 
-        analogWrite(PIN_MOSFET_B_LO, dutyCycle);
         analogWrite(PIN_MOSFET_B_HI, 0);
+        analogWrite(PIN_MOSFET_B_LO, dutyCycle);
```

Comment:
holy moly

Reply:
Yeah so you can see here if B_HI was high from the previous cycle, turning on B_LO would short 48v to Ground.